.github/workflows/integration.yml

```diff
@@ -111,32 +111,35 @@ jobs:
           retention-days: 3
 
   phpunit:
-    name: "${{ matrix.suite.name }} ${{matrix.suite.path }}"
+    name: "${{ matrix.suite.name }} ${{ matrix.suite.path }} ${{ matrix.db }}"
     continue-on-error: ${{ !matrix.stable }}
     runs-on: ubuntu-24.04
     strategy:
       fail-fast: false
       matrix:
         suite:
           - name: integration
-            path: tests/integration/Administration
+            path: Administration
           - name: integration
-            path: tests/integration/Core/Checkout
+            path: Core/Checkout
           - name: integration
-            path: tests/integration/Core/Content
+            path: Core/Content
           - name: integration
-            path: tests/integration/Core/Framework
+            path: Core/Framework
           - name: integration
-            path: tests/integration/Core/Installer
+            path: Core/Installer
           - name: integration
-            path: tests/integration/Core/Maintenance
+            path: Core/Maintenance
           - name: integration
-            path: tests/integration/Core/System
+            path: Core/System
           - name: integration
-            path: tests/integration/Elasticsearch
+            path: Elasticsearch
           - name: integration
-            path: tests/integration/Storefront
+            path: Storefront
           - name: migration,devops
+        db:
+          - mysql:8.0
+          - mariadb:10
```

Comment:
mariadb:11 does not work. The container does not get healthy

Reply:
maybe because of your health check, i think the binaries are renamed in that version